ports/boost-math/vcpkg.json

```
@@ -2,6 +2,7 @@
   "$comment": "Automatically generated by scripts/boost/generate-ports.ps1",
   "name": "boost-math",
   "version": "1.85.0",
+  "port-version": 1,
   "description": "Boost math module",
   "homepage": "https://github.com/boostorg/math",
```

```suggestion
  "homepage": "https://www.boost.org/libs/math",
```
as in #38814 (and already accepted in two cherry-picked PRs).
Then I might take `boost-math` out of #38814.
Or one of us rebases later.